ansible/roles/worker/tasks/setup_docker_driver.yml

```diff
@@ -36,3 +36,6 @@
   file:
     name=/images
     state=directory
+
+- name: Make a base directory for state files of the provisioned instances
```

Comment:
Hmm... or does the logging config also configure a default filelogger in some part of code that isn't changed? I'm a bit confused just by reading.